bootloader/ubootenv/env.go

```diff
@@ -33,12 +32,78 @@ import (
 	"strings"
 )
 
+// Env contains the data of the uboot environment stored in a given format
+type Env interface {
+	Get(string) string
+	Set(string, string)
+	Save() error
+
+	// mostly for tests or debugging
+	String() string
+	Size() int
+}
+
+// OpenFlags instructs open how to alter its behavior.
+type OpenFlags int
+
+// TODO: support "binary" format too?
+
+// Format is the uboot environment format
+type Format int
+
+const (
+	// OpenBestEffort instructs OpenWithFlags to skip malformed data without
+	// returning an error.
+	OpenBestEffort OpenFlags = 1 << iota
+
+	// OpenIgnoreComments instructs OpenWithFlags to read lines/entries that
+	// start with a "#" character, the same way mkenvimage does with input files
+	OpenIgnoreComments
```

Comment:
are there reasons for this one not to be the behavior always for the text format?

Reply:
not really, other than it's slightly complicated to have this be always on for the text format but not be always on for the native format

Reply:
isn't it a matter of making it unexported,  and doing flags|openIgnoreComment in openText ?

Reply:
sure that's fair I suppose